news/2020-02-24-mappers-report-february.md

```diff
@@ -0,0 +1,134 @@
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it through [this form](https://docs.google.com/forms/d/e/1FAIpQLSeVPi2KkcWiEsJzsEs1ir79HrP2jE_Sy2DCX_4nR_EECFTX4Q/viewform?usp=sf_link) for a chance of being featured on next month's installment of the Mappers' Report. We are looking to highlight more mapping contests from the community!
+
+### Monthly Beatmapping Contest: February 2020
+
+This month's installment of the Monthly Beatmapping Contest (MBC) has arrived! Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. If it's too late for you to make an entry given the deadline is in less than a week worry not, at the start of March a new contest will begin so you'll have your chance to participate. Keep your eyes peeled for the newspost which will be made when the March contest is ready.
+
+### 6th osu!Mapping Olympiad
+
+For the avid fruit catchers of the world, the 6th osu!Mapping Olympiad has shown itself, a great opportunity to flex your fruit placing muscles! This osu!catch contest requires you to map 2 difficulties for MYLK's Waffle Wave track. Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-15-osu-mapping-olympiad-6-catch) for all the details! As this is an olympiad the winner of the tournament will be given the title of Elite Mapper. Good luck to all of those participating, you have until March 13th to submit your fruity feasts.
+
+### Newspaper Cup
+
+The fantastic Chinese community organized tournament **Newspaper Cup** has made its yearly appearance and has now begun! This will be the contest's 6th yearly installment, making it one of the longest standing community organized mapping contests. All of the relevant information can be found [in this forum post](https://osu.ppy.sh/community/forums/topics/1023671) in both Chinese and English. You have until **March 15th** to submit your entries for one or more of the 4 available songs. Good luck to those planning on entering their creations into the contest!
+
+## <a id="summary"></a>Summary
+
+Make sure to tune in to the next NAT Meeting on March 1st in the [osu!dev Discord Server](https://discord.gg/ppy) and to do your best on all those fantastic mapping contests open for your submissions! Thank you for your time reading this quite hefty article. As this is our first installment of the Mappers' Report, any feedback is appreciated as we are still working hard at setting this up. Additionally, the next installments should be a bit smaller, as we had a lot to cover this time. We hope the information will prove itself useful to you!
```

Comment:
```suggestion
Make sure to tune in to the next NAT Meeting on March 1st in the [osu!dev Discord Server](https://discord.gg/ppy) and to do your best on all those fantastic mapping contests open for your submissions! Thank you for your time reading this quite hefty article. As this is our first installment of the Mappers' Report, any feedback is appreciated, as we are still working hard on setting this up. The next installments should be a bit smaller since we had a lot to cover this time. We hope the information will prove itself useful to you!
```